Docs for dataplane upgrade on k8s (#18051)
hashicorp/consul: website/content/docs/k8s/upgrade/index.mdx

1. Restart all gateways in your service mesh.

1. Now that all services and gateways are using Consul dataplanes, disable client agents in your Helm chart by deleting the `client` stanza or setting `client.enabled` to `false` and running a `consul-k8s` or Helm upgrade.

> 1. If `acls.manageSystemACLs` is set to `true`, outdated ACL tokens will persist a result of the upgrade. You can manually delete the tokens to declutter your Consul environment.

Reviewer: ```suggestion
1. If ACLs are enabled, outdated ACL tokens will persist a result of the upgrade. You can manually delete the tokens to declutter your Consul environment.
```

1. Restart all gateways in your service mesh.

1. Now that all services and gateways are using Consul dataplanes, disable client agents in your Helm chart by deleting the `client` stanza or setting `client.enabled` to `false` and running a `consul-k8s` or Helm upgrade.

1. If you have ACLs enabled, you will have some old ACL tokens that are now no longer needed. If you wish, you can manually clean up these tokens.

>    The old connect-injector tokens can be identified by the description `token created via login: {"component":"connect-injector"}`. Note that you should not delete
>    the tokens that have a description with `pod` as a key (e.g. `token created via login: {"component":"connect-injector","pod":"default/consul-connect-injector-576b65747c-9547x"}`) as those
>    are the tokens used by the new dataplane-enabled connect inject pods.
> 
>    You can also look at the creation date for the tokens and only delete the injector tokens created before your upgrade (do not delete all old tokens as some, e.g. the server tokens, are still in use).

Reviewer 2: ```suggestion
   Outdated connect-injector tokens have the following description: `token created via login: {"component":"connect-injector"}`. Do not delete
   the tokens that have a description where `pod` is a key, for example `token created via login: {"component":"connect-injector","pod":"default/consul-connect-injector-576b65747c-9547x"}`). The dataplane-enabled connect inject pods use these tokens.    

   You can also review the creation date for the tokens and only delete the injector tokens created before your upgrade, but do not delete all old tokens without considering if they are still in use. Some tokens, such as the server tokens, are still necessary.
```
Switch to active voice
